.github/workflows/deploy_to_qa.yaml

```diff
@@ -0,0 +1,108 @@
+name: 'Deploy to QA'
+on:
+  push:
+    branches:
+      - 'master'
+  
+jobs:
+  deploy_to_qa:
+    name: 'Deploy to AQ'
```

Comment:
Liten skrivefeil

Reply:
Thanks - I'll include the typo fix in the next PR extending the actions